app/models/attachment.rb

```diff
@@ -34,6 +34,21 @@ def associated_attachment
     attachable.attachments.find_by(id: metadata['associated_attachment_id'])
   end
 
+  def nonzipped_file_extension
```

Comment:
I don't think this is necessary, see comment in concatenation_service.

Reply:
Removed as it's no longer necessary in 0edf68ed95c38f320ab27504b13a339494c776ce